jax/version.py

```diff
@@ -13,5 +13,7 @@
 # limitations under the License.
 
 __version__ = "0.3.5"
+__version_info__ = tuple(int(i) for i in __version__.split(".") if i.isdigit())
 
 _minimum_jaxlib_version = "0.3.0"
+_minimum_jaxlib_version_info = tuple(int(i) for i in _minimum_jaxlib_version.split(".") if i.isdigit())
```

Comment:
Instead of repeating this twice, maybe make a private function?

Reply:
Done.